config/defaults.go

```diff
@@ -0,0 +1,308 @@
+	AuthRequireSigningDefault  = false
+	AuthSchemeDefault          = "xpub"
+	AuthSigningDisabledDefault = true
+)
+
+// Beef defaults
+const (
+	UseBeefDefault                  = true
+	PulseHeaderValidationURLDefault = "http://localhost:8000/api/v1/chain/merkleroot/verify"
+	PulseAuthTokenDefault           = "asd"
+)
+
+// Cachestore defaults
+const (
+	CacheEngineDefault = "freecache"
+)
+
+// Cluster defaults
+const (
+	ClusterCoordinatorDefault         = "redis"
+	ClusterPrefixDefault              = "bux_cluser_"
+	ClusterRedisURLDefault            = "localhost:6379"
+	ClusterRedisMaxIdleTimeoutDefault = "10s"
+	ClusterRedisUseTLSDefault         = false
+)
+
+// Datastore defaults
+const (
+	DatastoreAutoMigrateDefault = true
+	DatastoreDebugDefault       = false
+	DatastoreEngineDefault      = "sqlite"
+	DatastoreTablePrefixDefault = "xapi"
+)
+
+// MongoDB config keys
+const (
+	MongoDatabaseNameDefault = "xapi"
+	MongoTransactionsDefault = false
+	MongoURIDefault          = "mongodb://localhost:27017/xapi"
+)
+
+// SQL (MySQL, PostgreSQL) config keys
+const (
+	SQLDriverDefault                    = "postgresql"
+	SQLHostDefault                      = "localhost"
+	SQLNameDefault                      = "xapi"
+	SQLPasswordDefault                  = ""
+	SQLPortDefault                      = "5432"
+	SQLReplicaDefault                   = false
+	SQLSkipInitializeWithVersionDefault = true
+	SQLTimeZoneDefault                  = "UTC"
+	SQLTxTimeoutDefault                 = "10s"
+	SQLUserDefault                      = "postgres"
+)
+
+// SQLite config keys
+const (
+	SQLiteDatabasePathDefault = "./test-json.db"
+	SQLiteSharedDefault       = true
+)
+
+// Graphql defaults
+const (
+	GraphqlEnabledDefault        = true
+	GraphqlPlaygroundPathDefault = "/graphql"
+	GraphqlServerPathDefault     = "/graphql"
+)
+
+// Monitor defaults
+const (
+	MonitorAuthTokenDefault                   = ""
+	MonitorBuxAgentURLDefault                 = "ws://localhost:8000/websocket"
+	MonitorDebugDefault                       = false
+	MonitorEnabledDefault                     = false
+	MonitorFalsePositiveRateDefault           = 0.01
+	MonitorLoadMonitoredDestinationsDefault   = false
+	MonitorMaxNumberOfDestinationsDefault     = 100000
+	MonitorMonitorDaysDefault                 = 7
+	MonitorProcessorTypeDefault               = "bloom"
+	MonitorSaveTransactionDestinationsDefault = true
+)
+
+// NewRelic defaults
+const (
+	NewRelicDomainNameDefault = "domain.com"
+	NewRelicEnabledDefault    = false
+	NewRelicLicenseKeyDefault = "BOGUS-LICENSE-KEY-1234567890987654321234"
+)
+
+// Nodes defaults
+const (
+	NodesUseMapiFeeQuotesDefault = true
+	NodesMinercraftAPIDefault    = "mAPI"
+)
+
+// NodesBroadcastClientAPIsDefault is the default template of broadcast-client api tokens
+var NodesBroadcastClientAPIsDefault = []string{"url|token"}
```

Comment:
I'm not sure (please check) but using "url|token" was enabling go-broadcast client and it was trying to call for query/broadcast using:
`url/tx/xyz Bearer: token`
@pawellewandowski98 do you remember?

Reply:
Yes, and then there was an error from go-broadcast client that the URL is malformed. 
IMHO we need to talk with BA what should be the default for Arc API url

Reply:
Okay, for now I removed it completely and left an empty array []